pom.xml

```diff
@@ -71,6 +74,16 @@
 			<artifactId>jakarta.persistence-api</artifactId>
 			<version>2.2.3</version>
 		</dependency>
+		<dependency>
+			<groupId>io.jsonwebtoken</groupId>
+			<artifactId>jjwt</artifactId>
+			<version>0.9.0</version>
+		</dependency>
+		<dependency>
```

Comment:
We should remove this lombok dependency. Its great for usage, but for internship, we should not use it.